src/ddb/ddb_vos.c

```diff
@@ -1636,3 +1635,60 @@ dv_sync_smd(dv_smd_sync_complete complete_cb, void *cb_args)
 
 	return rc;
 }
+
+struct vea_cb_args {
+	dv_vea_extent_handler	 vca_cb;
+	void			*vca_cb_args;
+};
+
+static int
+vea_free_extent_cb(void *cb_arg, struct vea_free_extent *vfe)
+{
+	struct vea_cb_args	*args = cb_arg;
+
+	if (args->vca_cb)
+		return args->vca_cb(args->vca_cb_args, vfe);
+
+	return 0;
+}
+
+int
+dv_enumerate_vea(daos_handle_t poh, dv_vea_extent_handler cb, void *cb_arg)
+{
+	struct vea_cb_args	 args = {.vca_cb = cb, .vca_cb_args = cb_arg};
+	struct vos_pool		*pool;
+	struct vea_space_info	*vsi;
+	int			 rc;
+
+	pool = vos_hdl2pool(poh);
+	vsi = pool->vp_vea_info;
```

Comment:
This is defined in vos_internal.h, should we export it as well? Maybe in follow-on patches.

Reply:
Ya, I haven't removed completely the dependency on vos_internal. I will make sure there's a ticket to resolve this.